docs/api/relay.md

```diff
@@ -41,34 +41,19 @@ One of these options must be specified when instantiating a node supporting the
 ## Receiving messages
 ```go
 ...
-sub, err := wakuNode.Relay().Subscribe(context.Background())
+contentFilter := protocol.NewContentFilter(relay.DefaultWakuTopic)
+sub, err := wakuNode.Relay().Subscribe(context.Background, contentFilter) ([]*Subscription, error)
 if err != nil {
     fmt.Println(err)
     return
 }
 
-for value := range sub.C {
+for value := range sub[0].C {
     fmt.Println("Received msg:", string(value.Message().Payload))
 }
 ...
 ```
-To receive messages sent via the relay protocol, you need to subscribe to a pubsub topic. This can be done via any of these functions:
-- `wakuNode.Relay().Subscribe(ctx)` - subscribes to the default waku pubsub topic `/waku/2/default-waku/proto`
-- `wakuNode.Relay().SubscribeToTopic(ctx, topic)` - subscribes to a custom pubsub topic
-
-These functions return a `Subscription` struct containing a channel on which messages will be received. To stop receiving messages in this channel `sub.Unsubscribe()` can be executed which will close the channel (without unsubscribing from the pubsub topic)
-
-> Pubsub topics should follow the [recommended usage](https://rfc.vac.dev/spec/23/) structure. For this purpose, the `NewPubsubTopic` helper function was created:
-```go
-import 	"github.com/waku-org/go-waku/waku/v2/protocol"
-
-topic := protocol.NewPubsubTopic("the_topic_name", "the_encoding")
-/*
-fmt.Println(topic.String())  // => `/waku/2/the_topic_name/the_encoding`
-*/
-```
-
-
+To receive messages sent via the relay protocol, you need to subscribe specifying a content filter with the function `Subscribe(ctx context.Context, contentFilter waku_proto.ContentFilter, opts ...RelaySubscribeOption) ([]*Subscription, error)`. This functions return a list of `Subscription` struct containing a channel on which messages will be received. To stop receiving messages in this channel `sub.Unsubscribe()` can be executed which will close the channel (without unsubscribing from the pubsub topic)
```

Comment:
We should ask users to use relay.ubsubscribe rather than sub.Unsubscribe. 
This will make sure subscription is stopped i.e messages not sent to subscription, and if no other subscriptions exist for underlying pubsub topic, the pubsub is also unsubscribed.